src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md

```diff
@@ -20,7 +20,7 @@ This article gives a comprehensive description of each layout file type.
 
 ## Page layout {#layout-types-page}
 
-Page layout declares the wireframe of a page inside the `<body>` section, for example one-column layout or two-column layout.
+Page layout declares the wireframe of a page inside the `<body>` section, for example, one-column layout or two-column layout.
```

Comment:
Normally use of 2 commas like this means the section between the commas can be "removed" and the sentence still makes sense. Not the case here, so full stop looks better
```suggestion
Page layout declares the wireframe of a page inside the `<body>` section. For example, one-column layout or two-column layout.
```

Reply:
Thanks for your notes!
Should I revert my changes here?

Reply:
Hi - if it is quicker then that's what we normally do :)

Reply:
Done!

Reply:
@BarnyShergold  all your suggestions have been implemented. Could you please review.